test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -14,20 +14,74 @@
 // limitations under the License.
 // </copyright>
 
+using System;
 using System.Collections.Generic;
 using System.Diagnostics.Metrics;
-using System.Threading.Tasks;
 using Xunit;
 
-#nullable enable
-
 namespace OpenTelemetry.Metrics.Tests
 {
     public class MetricApiTest
```

Comment:
nit: I'd suggest to create a new class named HistogramAggregatorTest as this is purely testing the HistogramAggregator and nothing else.